netbout-web/src/main/java/com/netbout/rest/MarkdownTxtmark.java

```diff
@@ -47,10 +47,22 @@
         // @checkstyle LineLengthCheck (1 line)
         "(?<!\\]\\()(?<!=\")(https?:\\/\\/[a-zA-Z0-9-._~:\\?#@!$&'*+,;=%\\/]+[a-zA-Z0-9-_~#@$&'*+=%\\/])(?![\\w.]*\\]\\()"
     );
+    /**
+     * Pattern to detect lines which should have a line break on the end.
+     * We look for lines which have less then two spaces on their end because
+     * TxtMark automatically puts <br/> if the line has two or more spaces
```

Comment:
@dmzaytsev very minor: should be `{@code <br/>}`, otherwise we have just line break when Javadoc is rendered